src/devices/adeo.ts

```diff
@@ -410,6 +410,15 @@ const definitions: Definition[] = [
             await reporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['ZB-SMART-PIRTH-V3'],
+        model: 'ZB-SMART-PIRTH-V3',
```

Comment:
Could you update this with the model number which is written on the device/product page?